app/src/main/java/com/example/tungcon/demo/movie/MovieAdapter.java

```diff
@@ -0,0 +1,94 @@
+package com.example.tungcon.demo.movie;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.CardView;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.RatingBar;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.example.tungcon.demo.R;
+import com.example.tungcon.demo.data.model.Movie;
+
+
+import java.util.List;
+
+public class MovieAdapter extends RecyclerView.Adapter<MovieAdapter.ViewHolder> {
+    private Context mContext;
+    private List<Movie> mMovieList;
+    OnImageClickListener mOnImageClickListener;
```

Comment:
missing access modifier 
```suggestion
    private OnImageClickListener mOnImageClickListener;
```